query/branch/default_branch_requires_code_owners_review.sql

```diff
@@ -0,0 +1,13 @@
+select
+  -- Required Columns
+    r.full_name as resource,
+    case
+      when b.required_pull_request_reviews is not null and (b.required_pull_request_reviews -> 'require_code_owner_reviews')::bool is true  then 'ok'
+      else 'alarm'
+    end as status,
+      r.full_name || ' default branch ' || r.default_branch || case when(b.required_pull_request_reviews is not null and (b.required_pull_request_reviews -> 'require_code_owner_reviews')::bool is true) then ' enforces code owner''s review.' else ' does not enforce code owner''s review.' end as reason,
```

Comment:
can we update the format as per - https://github.com/turbot/steampipe-mod-github-compliance/pull/4/files#r946451106